tests/FiberTraces.py

```diff
@@ -854,10 +794,12 @@ def suite():
 
     suites = []
     suites += unittest.makeSuite(FiberTraceTestCase)
-#    suites += unittest.makeSuite(tests.MemoryTestCase)
     return unittest.TestSuite(suites)
 
 def run(exit = False):
+    """Quiet down loggers which are too verbose"""
+    lsst.log.setLevel("CameraMapper", lsst.log.FATAL)
+    lsst.log.setLevel("afw.image.ExposureInfo", lsst.log.FATAL)
```

Comment:
These are the outputs:

INFO  CameraMapper: Loading registry registry from /Users/azuri/stella-git/drp_stella_data/tests/data/PFS/registry.sqlite3
INFO  CameraMapper: Loading calibRegistry registry from /Users/azuri/stella-git/drp_stella_data/tests/data/PFS/CALIB/calibRegistry.sqlite3
INFO  afw.image.ExposureInfo: Empty WCS extension, using FITS header
INFO  afw.image.ExposureInfo: Empty WCS extension, using FITS header

I have not filed a ticket upstream yet but can do